areader/reader.go

```diff
@@ -841,7 +841,7 @@ func (ar *Reader) assignUpdateFiles() error {
 
 // should be `headers/0000/file` format
 func getUpdateNoFromHeaderPath(path string) (int, error) {
-	split := strings.Split(path, string(os.PathSeparator))
+	split := strings.Split(path, string("/"))
```

Comment:
Hey @h0ru5, so this is the actual fix here, correct? The rest is just stuff to make it locally work for you?

Reply:
yes, correct.
 thought I forked 2 different branches to upstream and to run locally, sorry for the confusion

Reply:
ok, realized, i put the PR on the wrong branch. the changhe you refer to is the only needed